<html></head><body><title>Exception Stack Trace</title>java.lang.AssertionError:  Age Validation in response expected [57] but found [58]<br>	at org.testng.Assert.fail(Assert.java:93)<br>	at org.testng.Assert.failNotEquals(Assert.java:512)<br>	at org.testng.Assert.assertEqualsImpl(Assert.java:134)<br>	at org.testng.Assert.assertEquals(Assert.java:115)<br>	at org.testng.Assert.assertEquals(Assert.java:388)<br>	at com.test.validators.assertions.CustomAssert.assertEquals(CustomAssert.java:50)<br>	at com.test.validators.api.AgeValidator.validateGetAgeResponse(AgeValidator.java:28)<br>	at systemTesting.agePredictor.AgePredictionTest.validateGetAgeAPi(AgePredictionTest.java:28)<br>	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)<br>	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)<br>	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)<br>	at java.lang.reflect.Method.invoke(Method.java:498)<br>	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)<br>	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)<br>	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)<br>	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)<br>	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)<br>	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)<br>	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)<br>	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)<br>	at java.lang.Thread.run(Thread.java:748)<br>